src/main/java/com/artipie/Users.java

```diff
@@ -188,7 +188,7 @@ public FromStorageYaml(final Storage storage, final Key key) {
                         for (final String group : user.ugroups) {
                             seq = seq.add(group);
                         }
-                        info.add(FromStorageYaml.GROUPS, seq.build());
+                        info = info.add(FromStorageYaml.GROUPS, seq.build());
```

Comment:
@olenagerasimova it seems we lack test coverage for adding groups, I think we should update `UsersFromStorageYamlTest` to test it